src/parameterized_tendencies/radiation/RRTMGPInterface.jl

```diff
@@ -915,8 +915,12 @@ function _RRTMGPModel(
             # Area ratio appears in denominator of RRTMGP scaling functions, 
             # we therefore pass the multiplicative inverse from ClimaAtmos to 
             # use mult ops instead of div in RRTMGP GPU kernels.
+            z_max = maximum(z_lev)
```

Comment:
```suggestion
            z_max = Spaces.z_max(axes(z_lev))
```
Can we not use this instead? calling `maximum(z_lev)` will launch an additional global reduction kernel.

Reply:
Yes, thank you!

Reply:
Actually, `z_lev` is just an array so `Spaces.z_max` doesn't work.

Reply:
Do we not have any spaces / fields at this part in the code?

Reply:
If we don't then maybe we should?

Reply:
I don't think we have any space/field access since this is in `_RRTMGPModel` but we could definitely just pass a space or `z_max` in.